pages/book-a-demo.vue

```diff
@@ -0,0 +1,336 @@
+              class="bg-secondary text-white py-1.5 px-3 rounded-lg text-xl "
+            >
+              Submit
+            </button>
+          </div>
+          <div
+            v-if="errorMessage"
+            class="text-red-500 mt-6 inline-block error-message"
+          >
+            Something went wrong and the form was not submitted.
+            Please stay on this page and try again in a few minutes.
+          </div>
+        </div>
+      </form>
+    </div>
+    <Footer />
+  </div>
+</template>
+<script setup>
+// carousel icon
+import circle from "~/assets/icons/circle.svg";
+import circleActive from "~/assets/icons/circle-active.svg";
+const { $contentfulClient } = useNuxtApp()
+const activeCarouselImageIndex = ref(1);
+
+const myForm = ref(null)
+const errorMessage = ref(false);
+
+useSeoMeta({
+  title: 'Book a Demo'
+})
+
+onMounted(() => {
+  setInterval(() => {
+    if (
+      activeCarouselImageIndex.value === 0 ||
+      activeCarouselImageIndex.value === 3
+    ) {
+      activeCarouselImageIndex.value = 1;
+    } else {
+      activeCarouselImageIndex.value += 1;
+    }
+  }, 7000);
+});
+
+const pendingPage = ref(true)
+const pageData = await $contentfulClient.getEntries({
+  order: '-sys.createdAt',
+  content_type: 'getInvolved',
+  'metadata.tags.sys.id[in]': 'homePage'
+}).then((pageData) => {
+  pendingPage.value = false
+  return pageData.items[0];
+}).catch(console.error);
+
+const brandOne = pageData?.fields.brandImage1.fields.file.url
+const brandTwo = pageData?.fields.brandImage2.fields.file.url
+const brandThree = pageData?.fields.brandImage3.fields.file.url
+
+const activeCarouselImagePath = computed(() => {
+  const imageMap = {
+    1:brandOne, 
+    2:brandTwo, 
+    3:brandThree,
+  };
+  return imageMap[activeCarouselImageIndex.value];
+});
+const industries = [
+  'Agriculture & Forestry',
+  'Animals & Pets',
+  'Arts & Craft',
+  'Beauty & Fashion',
+  'Civic Engagement',
+  'Education',
+  'Entertainment & Film',
+  'Environment',
+  'Family & Parenting',
+  'Finance',
+  'Government & Public Sector',
+  'Health & Wellness',
+  'Hospitality & Tourism',
+  'Lifestyle',
+  'Media',
+  'Music',
+  'Nature',
+  'Nonprofit & Social Services',
+  'Professional Services',
+  'Sports & Outdoor',
+  'Technology',
+  'Other'
+];
+
+const selectedIndustry = ref('');
+
+function handleSubmit(e) {
+  e.preventDefault();
+
+  const formElement = myForm.value;
+  const formData = new FormData(formElement);
+
+  fetch("/", {
+    method: "POST",
+    headers: { "Content-Type": "application/x-www-form-urlencoded" },
+    body: new URLSearchParams(formData).toString(),
+  })
+    .then(() => {
+      navigateTo({
+        path: '/demo-success/',
+        query: {
+          
+        }
+      })
+    }).catch(() => errorMessage.value = true);
+}
+</script>
+
+<style>
+label {
+  color: #343538;
+  font-family: Poppins;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 20px;
+}
```

Comment:
```suggestion
label {
  font-weight: 600;
}
```

`font-weight` was the only style that had any effect on the labels here